docs/configuring-kubelinter.md

```diff
@@ -62,3 +62,59 @@ example,
 > [!TIP]
 > `exclude` always takes precedence, if you include and exclude the same check,
 > KubeLinter always skips the check.
+
+## Run custom checks
+
+You can write custom checks that are based on existing templates.
+
+For example,
+- To be sure that an annotation exists, you can add the following:
+  ```yaml
+  customChecks:
+    - name: required-annotation-responsible
+      template: required-annotation
+      params:
+        key: company.io/responsible
+  ```
+
+  It uses the existing `required-annotation` template and adds the check for your custom keys.
+
+- To be sure that a specific label exists, you can add the following:
+  ```yaml
+  customChecks:
+    - name: required-label-release
+      template: required-label
+      params:
+        key: company.io/release
+  ```
+
+  It uses the existing `required-label` template.
+
+### Extending custom checks
+
+There may be situations where custom checks need to be scoped to specific Kubernetes objects. To make custom checks more user friendly, a message can be added to the output of the check. These can be added by using `scope` or `remediation` respectively.
+
+For example,
+- To be sure that only Deployments have set a required annotation, you can add the following:
+  ```yaml
+  customChecks:
+    - name: required-annotation-responsible
+      template: required-annotation
+      params:
+        key: company.io/responsible
+      scope:
+        objectKinds:
+          - DeploymentLike
+  ```
+
+  At the time of this writing, `objectKinds` can be either `DeploymentLike` or `Any`.
```

Comment:
In fact we also support `Service` as well. Maybe instead of enumerating all the object kinds here, we can insert a link to this folder https://github.com/stackrox/kube-linter/tree/main/internal/objectkinds and say all supported object kinds can be found here?

Reply:
Sound muchs better, yes :+1: